monai/transforms/utility/dictionary.py

```diff
@@ -443,6 +448,38 @@ def inverse(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
         return d
 
 
+class EnsureTensord(MapTransform, InvertibleTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.EnsureTensor`.
+    """
+
+    def __init__(self, keys: KeysCollection, allow_missing_keys: bool = False) -> None:
+        """
+        Args:
+            keys: keys of the corresponding items to be transformed.
+                See also: :py:class:`monai.transforms.compose.MapTransform`
+            allow_missing_keys: don't raise exception if key is missing.
+        """
+        super().__init__(keys, allow_missing_keys)
+        self.converter = EnsureTensor()
+
+    def __call__(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
+        d = dict(data)
+        for key in self.key_iterator(d):
+            self.push_transform(d, key)
+            d[key] = self.converter(d[key])
+        return d
+
+    def inverse(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
+        d = deepcopy(dict(data))
+        for key in self.key_iterator(d):
+            # Apply inverse
+            d[key] = tensor_to_numpy(d[key])
```

Comment:
The inverse isn't necessarily back to numpy, it depends on the input type.

Reply:
Hi @rijobro ,

Here we only convert Tensor back to numpy, not all kinds of data, if scalar value, convert to regular number. Just the inverse operation of this transform.

Thanks.